src/RedshiftBase.php

```diff
@@ -391,19 +391,19 @@ private function dedup($inputTempTableName, $columns, array $primaryKey)
             return $this->quoteIdentifier($column);
         }, $columns));
 
-        $tempTable = $this->createTemporaryTableFromSourceTable($inputTempTableName, $primaryKey);
+        $tempTable = $this->createTemporaryTableFromDestinationTable($inputTempTableName, $primaryKey);
 
         $this->query("INSERT INTO {$this->tableNameEscaped($tempTable)} ($columnsSql) " . $sql);
         $this->replaceTempTables($tempTable, $inputTempTableName);
     }
 
-    private function createTemporaryTableFromSourceTable($sourceTableName, array $primaryKey, $schemaName = null)
+    private function createTemporaryTableFromDestinationTable($targetTableName, array $primaryKey, $schemaName = null)
```

Comment:
Tohle přejmenování mě dost zmátlo. `createTemporaryTableFromDestinationTable` mi moc nedává smysl.
Když už tak bych to přejmenoval na `createTemporaryTableFromSourceTable($sourceTableName, array $primaryKey, $schemaName = null)` at je to konzistentni s `createTableFromSourceTable`

Reply:
Jo počkat ono se to tak jmenovalo předtím. Tak bych to nechal, vystihuje to imho přesně co to dělá.

Reply:
Nebo jestli ti tam vádí to slovo source protože se to motá se source a destination. Tak to může oboje být `createTemporaryTableFromTable` a `createTableFromTable` :)

Reply:
Ty metody jsou private, takze nikde mimo `RedshiftBase` se nepouzivaj. A v kontextu toho jak jsou pouzity je ten nazev trochz zmatecny. Prejmenuju to tak jak rikas.